### ISSUE TITLE
Effect: Bad parameters 'Action' should be 'dynamic'

### ISSUE THREAD

**Issue by huextrat** [alibaba/fish-redux#372]
**Effect function with bad parameters type**

Since the last stable update of Flutter, I'm facing a problem caused by Action. I now that flutter added a new class 'Action' and we have to use : `hide action` in `material.dart`.

But now we have another error in the `_init` for instance and all the effects 

```dart
import 'package:fish_redux/fish_redux.dart';
import 'package:flutter/material.dart' hide Action;

Effect<LoginState> buildEffect() {
  return combineEffects(<Object, Effect<LoginState>>{
    Lifecycle.initState: _init
  });
}
``` 
```dart
void _init(Action action, Context<LoginState> ctx) async {
}
``` 

The error message is on the `_init`

![Capture d’écran 2019-07-09 à 11 46 10](https://user-images.githubusercontent.com/22995315/60878761-aae12800-a240-11e9-998e-17c9ac6174ae.png)

The error message say that we have to use a `dynamic` type instead of `Action` 

**To Reproduce**
Upgrade Flutter version to the latest stable and look at your effect file.

**Comment by huextrat** [comment]
Resolved in 0.2.3 !

**Comment by zjuwjf** [comment]
#303
